src/browser/extension/options/options.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Not sure it works on Windows, but my MacOS is also case insensitive and `git mv` to a random name then `git mv` back with upper case worked for me in the past. I can do it after merging the PR.